oak/server/dev/dev_oak_manager.h

```diff
@@ -39,12 +39,9 @@ class DevOakManager final : public Manager::Service {
   void InitializeAssertionAuthorities();
   std::string NewApplicationId();
 
-  // The port is listening on
-  int32_t port_;
   // The application id
   uint64_t application_id_;
   std::unordered_map<std::string, std::unique_ptr<OakRuntime>> runtimes;
-  std::unordered_map<std::string, std::unique_ptr<::grpc::Server>> servers;
```

Comment:
nit: Both `runtimes` and `servers` should have a [trailing underscore](https://google.github.io/styleguide/cppguide.html#Variable_Names).

Reply:
Done